src/components/NumberInput/_number-input.scss

```diff
@@ -0,0 +1,162 @@
+@import '../../globals/theme/ics-theme';
+@import 'carbon-components/src/components/number-input/number-input';
+@import '../../globals/type/ics-typography';
+@import '../UnderlineMotion/underline-motion';
+@import '../../globals/bidi/bidi';
+
+// Child atoms
+@import '../Form/form';
+
+@include exports('number-input--ics') {
+  .#{$prefix}--number-input__wrapper {
+    margin-top: rem(24px);
+    position: relative;
+    display: flex;
+    min-width: 200px;
+
+    .#{$prefix}--mi__underline {
+      position: absolute;
+      top: rem(40px);
+      @include bidi-right(rem(15px));
+      width: calc(100% - 16px);
+    }
+  }
+
+  .#{$prefix}--number {
+    @include reset;
+    @include type-scale-item(c, false);
+    box-sizing: border-box;
+    display: inline-flex;
+    width: 100%;
+    min-width: 9.375rem;
+    padding: 0;
+    height: auto;
+    color: $text-01;
+    background-color: transparent;
```

Comment:
Think this should be updated to use the field variable, which is now transparent. See ics-theme.scss